packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -221,6 +427,39 @@ export const deleteDBInstance = async (identifier: string, region: string): Prom
   }
 };
 
+/**
+ * Deletes the given RDS cluster and instances it contains
+ * @param identifier RDS cluster identifier to delete
+ * @param region RDS cluster region
+ */
+export const deleteDBCluster = async (identifier: string, region: string): Promise<void> => {
+  // First the instance deletion is triggered
+  const instanceID = createInstanceIdentifier(identifier);
+  console.log(`Deleting instance: ${instanceID}`);
+  await deleteDBInstance(instanceID, region);
+
+  // Now delete the cluster
+  const client = new RDSClient({ region });
+  const params: DeleteDBClusterCommandInput = {
+    DBClusterIdentifier: identifier,
+    SkipFinalSnapshot: true,
+  };
+  console.log(`Deleting cluster: ${identifier}`);
+  const command = new DeleteDBClusterCommand(params);
+  try {
+    await client.send(command);
+  } catch (error) {
+    console.log(error);
+    throw new Error(`Error in deleting RDS cluster: ${identifier}`);
```

Comment:
Should we include the body of `error`, rather than just logging it? The error message is useful for troubleshooting, and will be surfaced to the test runner rather than buried in logs (IIRC).